### PR TITLE
OCLOMRS-237: Add tooltip on the short code input field

### PR DESCRIPTION
**Pull request by emmabaye**
# JIRA TICKET NAME:
[Add tooltip on the short code input field](https://issues.openmrs.org/browse/OCLOMRS-237)

# Summary:
When adding a dictionary, "short code" input field should have a descriptive tooltip.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "react-router-dom": "^4.2.2",
     "react-scripts": "1.1.4",
     "react-select": "^2.0.0-beta.7",
-    "react-tooltip": "^3.6.0",
+    "react-tooltip": "^3.8.4",
```
Comment (alexmochu) [inline package.json]: Why did you change the version?

Comment (emmabaye) [reply package.json]: I didn't know react-tooltip was already installed. I'll revert to the previous version if it is okay.

**Comment by emmabaye** [comment]
> @emmabaye why has this taken 3 days without closing?

I'll get the team to reveiw it.

**Comment by dkayiwa** [comment]
@emmabaye can you ping me if they do not review by end of the day?

**Comment by emmabaye** [comment]
> @emmabaye can you ping me if they do not review by end of the day?

Alright.
